lms/lib/courseware_search/lms_result_processor.py

```diff
@@ -62,6 +62,14 @@ def url(self):
         )
 
     @property
+    def course_name(self):
```

Comment:
Optional: Might be nice to cache this result as with the others properties above... or ideally convert them all over to use @lazy decorator - just discovered this and it's super! (Thanks @e-kolpakov!)